examples/images_contours_and_fields/colormap_normalizations_symlognorm.py

```diff
@@ -5,21 +5,18 @@
 
 Demonstration of using norm to map colormaps onto data in non-linear ways.
 
+Here we plot data that has two humps, one negative and one positive.
+The positive hump has 5-times the amplitude of the negative.
+With a linear scale you cannot see detail in the negative hump.
+So we logarithmically scale the positive and negative data separately.
+
 .. redirect-from:: /gallery/userdemo/colormap_normalization_symlognorm
 """
 
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.colors as colors
 
-"""
```

Comment:
This should probably be a sphinx gallery block comment, which is in the form of a long line of `#`, followed by each line of the block starting with a `#`.

Reply:
I think that putting the description at the top is more consistent with other examples rather than adding a second paragraph after the imports.